### PR TITLE
Promise Support

### PR DESCRIPTION
**Pull request by eddiemoore**

### CODE REVIEW

File: test/swimsuit.js [hunk 1]
```diff
@@ -38,4 +38,17 @@ describe('Swimsuit', () => {
 
     expect(Swimsuit.test instanceof Promise).to.be.true
   })
+
+  it('should add support with a promise', (done) => {
+    const returnVal = 'Hello'
+    Swimsuit.supports('wait', () => {
+      return new Promise((resolve, reject) => {
```
Comment (sebdeckers) [inline test/swimsuit.js:45]: Does standard not allow single-statement arrow functions without the `{ ... }` anymore?

Comment (eddiemoore) [inline test/swimsuit.js:45]: As in

``` js
return new Promise((resolve, reject) => setTimeout(() => resolve(returnVal), 500))
```

?

Comment (sebdeckers) [inline test/swimsuit.js:45]: Never mind, guess it's required for multi-line.

**Comment by sebdeckers** [comment]
LGTM

**Pull request merged**